### DISCUSSION TITLE
Full screen error

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#352] [Q&A] [answered]
The display->full screen is set on "Standort folgen" and "Karte ansehen"  (Germany settings). When starting Follow mode full screen is ok. When leaving Follow mode the full screen is switched off. It will return to full screen when tapping on hamburger menu. Please have a look at it, thanks!

**Comment by kds112** [comment]
Nevertheless, the brand new beta is great!! 👍

**Comment by devemux86** [comment]
@kds112 thanks for the report!

It will work properly in the next update.

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
